docs/en_US/include/using-sileo.md

```diff
@@ -56,9 +56,9 @@ The tweak should now be installed to your device and will be active whenever you
 1. Tap "Remove"
     - A bar labeled "Queued" should appear at the bottom of your screen
 1. Tap on the bar and press "Confirm"
-    - It may take some time for the tweak to install
+    - It may take some time for the tweak to uninstall
 1. After it's finished, tap "<router-link to="/faq/#what-is-respringing">Respring</router-link>"
-    - The device should appear to reboot after this
+    - The device should appear to respring after this
```

Comment:
I'm unsure about this change (the average user probably won't immediately know the difference between a respring and a safely done shutdown/reboot).

If this change should be kept, imo the better way would be to just remove this line altogether.